dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/EventExporterTest.java

```diff
@@ -245,6 +250,54 @@ void testExportEventsWithLastUpdateDates() throws ForbiddenException, BadRequest
     assertContainsOnly(List.of("D9PbzJY8bJM"), events);
   }
 
+  @Test
+  void testExportEventsWithDatesIncludingTimeStamp()
+      throws ForbiddenException, BadRequestException {
+    Date date = new Date();
+    EventOperationParams params =
+        EventOperationParams.builder()
+            .orgUnitUid(orgUnit.getUid())
+            .enrollments(Set.of("nxP7UnKhomJ"))
+            .programStageUid(programStage.getUid())
+            .events(Set.of("pTzf9KYMk72"))
```

Comment:
If you are only after getting the `event` `pTzf9KYMk72` I would set `.events(Set.of("pTzf9KYMk72"))` on the builder. Setting more then that makes me question whether you are testing querying by dates but I don't think these are correct 
 The `pTzf9KYMk72` `"updatedAt": "2019-01-28T12:10:38.109"` which is not within the range you specify 🤔

Reply:
resolved